CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# [1.2.5] - 2019-02-21
+### Added
+* Hive 1.2 compatible `tableExists` method
```

Comment:
Add period at the end and a link to the CT issue (like the item on line 12).

Reply:
I think this is a bug that you are fixing, so, the title above should be ### Fixed at the top